README.md

```diff
@@ -108,6 +108,26 @@ Timing stats for 10 failed connections min/avg/max/dev = 5.00025s/5.001741s/5.00
 2
 ```
 
+## Executing the tests
```

Comment:
I wouldn't says "tests", because you are covering also other administrative tasks

Reply:
Mmm... i don't think so. I've renamed to "Testing locally", but in the end, everything in this section tries to provide hints to reproduce locally whatever travis does (essentially tests, but also the deployment you may want to run locally for testing purposes)

Reply:
👍